reference/5.1/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,139 @@
+---
+ms.date:  11/27/2018
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs internal operations from the engine, providers, and
+cmdlets.
+
+## Long Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the engine and starting and stopping providers. It will
+also log details about PowerShell commands.
+
+## Viewing the PowerShell event log on Windows
+
+PowerShell logs can be viewed using the Event Viewer. The event log is
+located in the Application and Services Logs group and is named `Microsoft-Windows-PowerShell`.
+The associated ETW provider `GUID` is `A0C1853B-5C40-4B15-8766-3CF1C58F985A}`.
+
+When script block logging is enabled,
+PowerShell will log the following events to the `Microsoft-Windows-PowerShell/Operational` log:
+
+|Field| Value|
+|-|-|
+|EventId|`4104` / `0x1008`|
+|Channel|`Operational`|
+|Level|`Verbose`|
+|Opcode|`Create`|
+|Task|`CommandStart`|
+|Keyword|`Runspace`|
+
+## Enabling Script Block Logging
+
+Any new PowerShell session will pickup the new setting after making one of the following changes.
+
+> [!NOTE]
+> It is recommended to enable Protected Event Logging when
+> using Script Block logging for anything other than diagnostics purposes.
+
+### Using Group Policy
+
+To enable automatic transcription,
+enable the ‘Turn on PowerShell Script Block Logging’
+feature in Group Policy through `Windows Components -> Administrative Templates -> Windows PowerShell`.
+
+### Using the Registry
+
+Run the following function:
+
+```powershell
+function Enable-PSScriptBlockLogging
+{
+    $basePath = “HKLM:\Software\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging”
+
+    if(-not (Test-Path $basePath))
+    {
+        $null = New-Item $basePath -Force
+    }
+
+    Set-ItemProperty $basePath -Name EnableScriptBlockLogging -Value “1”
+}
+```
+
+## Protected Event Logging
+
+One concern when increasing the amount of logging on a system
+is the danger that logged content may contain sensitive data.
+For example, if you log the content of every PowerShell script that was run,
+there is the possibility that a script may contain credentials or other sensitive data.
+
+If an attacker later compromises a machine that has logged this data,
+it may provide them with additional information with which to extend their reach.
+
+To prevent this dilemma,
+Windows 10 introduces Protected Event Logging.
```

Comment:
We should include a link to 'Windows 10 Proteced Event Logging'

Reply:
I cannot find a link.